### ISSUE TITLE
Max Upload Size

### ISSUE THREAD

**Issue by mason-chase** [microsoft/msstore-cli#42]
Hi Everyone,

I'm trying to upload an MSIX installer that is around 310mb and I'm getting below error when I use `msstore-cli` 
If I upload this file manually through the WEB UI at partner.microsoft.com it uploads without error.

```
06:18:55 fail: MSStore.CLI.ProjectConfigurators.ElectronProjectConfigurator[0] Error while uploading
the application package. MSStore.API.MSStoreException: <?xml version="1.0" %
encoding="utf-8"?><Error><Code>RequestBodyTooLarge</Code><Message>The request body is too large and
exceeds the maximum permissible limit.
RequestId:f8bbfb32-f01e-00ce-6fca-b1cde1000000
Time:2024-05-29T13:18:22.1807612Z</Message><MaxLimit>268435456</MaxLimit></Error>    at
MSStore.CLI.Services.AzureBlobManager.UploadFileAsync(String blobUri, String localFilePath,
IProgress`1 progress, CancellationToken ct) in /_/MSStore.CLI/Services/AzureBlobManager.cs:line 44
at
MSStore.CLI.Helpers.IStorePackagedAPIExtensions.<>c__DisplayClass10_0.<<PublishAsync>b__0>d.MoveNext
() in /_/MSStore.CLI/Helpers/IStorePackagedAPIExtensions.cs:line 441
Error while uploading the application package.
```

Any clue is highly appreciated.

Regards

**Comment by azchohfi** [comment]
Oh, interesting. We are using a simple httpclient to do the job here, so we are limited to 256mb. We need to either implement the batch or add a dependency on the Azure Blob library to that for us. I'll look into fixing this. Thanks for the report!

**Comment by azchohfi** [comment]
Should be fixed on v0.1.33.
